entries.go

```diff
@@ -218,3 +218,13 @@ func GetEntry(bodyByteGetterFunc BodyByteGetter, articleID int) (Item, error) {
 	err = json.Unmarshal(body, &item)
 	return item, err
 }
+
+// ExportEntry queries the API for a specific article content according to the API request /entries/ID
```

Comment:
I would suggest:
`// ExportEntry queries the API to retrieve a single entry in a predefined format according to the API request /entries/ID/export.FORMAT`